### ISSUE TITLE
SUPPORT: DAE Failure

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#3056]
ZOOM can not get any raw or good frames on the dashboard. Investigate, fix and write up solution.

**Comment by ChrisM-S** [comment]
It's still not clear exactly what happened here but the problem as we were presented with it when we went down to ZOOM was that there was no power on the detectors and therefore no input signals to the ADC cards - hence no counts in the DAE. There was someone working in the Detector tank which was why the power was off.

Useful things to check/know in this case were:

- [ ] Visit the beamline - (in this case also with electronics as we suspected a hardware problem).  Software usually doesn't just stop normally when other things are working  - right? :smile: 
- [ ] Most importantly, ask the scientists if anything happened around the time of the problem, in this case they mentioned someone had moved a cable on an ADC (although this was not the problem!).
- [ ] Simple check, look at the lights on the ADC cards, if no flickering, there is no data coming in and this is a good indicator that the HT might be off (a few lights might mean shutter closed or beam off).
- [ ] If frame/raw counts are not showing up, a good diagnostic is to put the DAE into "Internal Test Clock".  If this works and frames appear, it is likely that there may be a problem with a Time of Flight signal (this often affects more than one beamline.

My suspicion is that maybe the detector power was turned off sooner than then scientists expected for the work in the tank to be done!
